@navikt/core/react/src/inline-message/icon/InlineMessageIcon.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import {
+  CheckmarkCircleFillIcon,
+  ExclamationmarkTriangleFillIcon,
+  InformationSquareFillIcon,
+  XMarkOctagonFillIcon,
+} from "@navikt/aksel-icons";
+import { useRenameCSS } from "../../theme/Theme";
+import { useI18n } from "../../util/i18n/i18n.hooks";
+
+const VARIANT_ICONS = {
+  info: InformationSquareFillIcon,
+  success: CheckmarkCircleFillIcon,
+  warning: ExclamationmarkTriangleFillIcon,
+  error: XMarkOctagonFillIcon,
+} as const;
+
+function InlineMessageIcon({
+  variant,
+}: {
+  variant: "info" | "success" | "warning" | "error";
+}) {
+  const translate = useI18n("Alert");
+  const { cn } = useRenameCSS();
+
+  if (!variant) {
+    return null;
+  }
```

Comment:
Do we need this check?

```suggestion
```

Reply:
It just fails a little more gracefully, if not we get this when user forgets "variant"
<img width="739" height="80" alt="Screenshot 2025-10-22 at 10 32 11" src="https://github.com/user-attachments/assets/1ca0fdb3-e734-45ea-92d1-13768df8de14" />

Reply:
Consider changing to this to cover mistypings as well:
```suggestion

  if (!(variant in VARIANT_ICONS)) {
    return null;
  }
```